drivers/hmis/app/models/hmis/ce/referral_enroller.rb

```diff
@@ -51,6 +51,18 @@ def create_enrollment(message)
       referral.update!(target_enrollment: enrollment)
     end
 
+    def delete_wip_enrollment(_message)
+      enrollment = referral.target_enrollment
+      return unless enrollment
+
+      raise "target enrollment #{enrollment.id} has already had intake completed" unless enrollment.in_progress?
```

Comment:
Could be raised if provider completes Change Provider Outcome step after having already completed intake

Reply:
What will be the course of action to fix the referral when it gets in this state?

Perhaps we can skip the action and report to sentry (For now) to keep track of this, and come back to it and add a user-facing validation message later..

```suggestion
      unless enrollment.in_progress?
          message = "target enrollment #{enrollment.id} has already had intake completed, unable to perform delete_wip_enrollment message"
          raise message if Rails.env.development?

          Sentry.capture_message(message)
          return # in non-dev env: return, we are unable to perform the action
     end
```